# Exploratory Data Analysis - EDA

## COVID Clinical Trials

In [1]:
# Import libraries
import numpy as np
import pandas as pd

In [2]:
# Load the dataset
df = pd.read_csv("COVID clinical trials.csv")

# Display all columns of the dataset
pd.set_option('display.max_columns', None)

### 1. Explore the dataset by checking shape, columns, and see the first/last 'n' rows using head/tail. (n= 5,15,30)

In [3]:
# Shape of the dataset
df.shape

(5783, 27)

*There are **5783** rows and **27** columns in the dataset.*

In [4]:
# Columns in the dataset
df.columns

Index(['Rank', 'NCT Number', 'Title', 'Acronym', 'Status', 'Study Results',
       'Conditions', 'Interventions', 'Outcome Measures',
       'Sponsor/Collaborators', 'Gender', 'Age', 'Phases', 'Enrollment',
       'Funded Bys', 'Study Type', 'Study Designs', 'Other IDs', 'Start Date',
       'Primary Completion Date', 'Completion Date', 'First Posted',
       'Results First Posted', 'Last Update Posted', 'Locations',
       'Study Documents', 'URL'],
      dtype='object')

In [5]:
# First 5 rows of the dataset
df.head()

Rank   NCT Number                                              Title  \
0     1  NCT04785898  Diagnostic Performance of the ID Now™ COVID-19...   
1     2  NCT04595136  Study to Evaluate the Efficacy of COVID19-0001...   
2     3  NCT04395482  Lung CT Scan Analysis of SARS-CoV2 Induced Lun...   
3     4  NCT04416061  The Role of a Private Hospital in Hong Kong Am...   
4     5  NCT04395924         Maternal-foetal Transmission of SARS-Cov-2   

        Acronym                  Status         Study Results  \
0   COVID-IDNow  Active, not recruiting  No Results Available   
1      COVID-19      Not yet recruiting  No Results Available   
2   TAC-COVID19              Recruiting  No Results Available   
3      COVID-19  Active, not recruiting  No Results Available   
4  TMF-COVID-19              Recruiting  No Results Available   

                                          Conditions  \
0                                            Covid19   
1                               SARS-CoV-2 Infection   
2                                            covid19   
3                                              COVID   
4  Maternal Fetal Infection Transmission|COVID-19...   

                                       Interventions  \
0   Diagnostic Test: ID Now™ COVID-19 Screening Test   
1    Drug: Drug COVID19-0001-USR|Drug: normal saline   
2  Other: Lung CT scan analysis in COVID-19 patients   
3          Diagnostic Test: COVID 19 Diagnostic Test   
4  Diagnostic Test: Diagnosis of SARS-Cov2 by RT-...   

                                    Outcome Measures  \
0  Evaluate the diagnostic performance of the ID ...   
1  Change on viral load results from baseline aft...   
2  A qualitative analysis of parenchymal lung dam...   
3  Proportion of asymptomatic subjects|Proportion...   
4  COVID-19 by positive PCR in cord blood and / o...   

                               Sponsor/Collaborators  Gender  \
0              Groupe Hospitalier Paris Saint Joseph     All   
1                         United Medical Specialties     All   
2                       University of Milano Bicocca     All   
3                    Hong Kong Sanatorium & Hospital     All   
4  Centre Hospitalier Régional d'Orléans|Centre d...  Female   

                                         Age           Phases  Enrollment  \
0  18 Years and older   (Adult, Older Adult)   Not Applicable      1000.0   
1  18 Years and older   (Adult, Older Adult)  Phase 1|Phase 2        60.0   
2  18 Years and older   (Adult, Older Adult)              NaN       500.0   
3                  Child, Adult, Older Adult              NaN      2500.0   
4             18 Years to 48 Years   (Adult)              NaN        50.0   

  Funded Bys      Study Type  \
0      Other  Interventional   
1      Other  Interventional   
2      Other   Observational   
3   Industry   Observational   
4      Other   Observational   

                                       Study Designs         Other IDs  \
0  Allocation: N/A|Intervention Model: Single Gro...       COVID-IDNow   
1  Allocation: Randomized|Intervention Model: Par...  COVID19-0001-USR   
2  Observational Model: Cohort|Time Perspective: ...       TAC-COVID19   
3  Observational Model: Cohort|Time Perspective: ...        RC-2020-08   
4  Observational Model: Cohort|Time Perspective: ...      CHRO-2020-10   

         Start Date Primary Completion Date   Completion Date  \
0  November 9, 2020       December 22, 2020    April 30, 2021   
1  November 2, 2020       December 15, 2020  January 29, 2021   
2       May 7, 2020           June 15, 2021     June 15, 2021   
3      May 25, 2020           July 31, 2020   August 31, 2020   
4       May 5, 2020                May 2021          May 2021   

       First Posted Results First Posted Last Update Posted  \
0     March 8, 2021                  NaN      March 8, 2021   
1  October 20, 2020                  NaN   October 20, 2020   
2      May 20, 2020                  NaN   November 9, 2020   
3      June 4, 2020  

In [6]:
# Last 5 rows of the dataset
df.tail()

Rank   NCT Number                                              Title  \
5778  5779  NCT04011644  Mobile Health for Alcohol Use Disorders in Cli...   
5779  5780  NCT04681339  Antibiotic Prescription in Children Hospitaliz...   
5780  5781  NCT04740229  Moderate-intensity Flow-based Yoga Effects on ...   
5781  5782  NCT04804917           3-year Follow-up of the Mind My Mind RCT   
5782  5783  NCT04680000  Chronic Pain Management In Primary Care Using ...   

           Acronym              Status         Study Results  \
5778           NaN          Recruiting  No Results Available   
5779           NaN  Not yet recruiting  No Results Available   
5780           NaN          Recruiting  No Results Available   
5781  MindMyMindFU          Recruiting  No Results Available   
5782           NaN  Not yet recruiting  No Results Available   

                                             Conditions  \
5778                      Alcohol Drinking|Telemedicine   
5779  Community Acquired Pneumonia in Children|Antib...   
5780                               Stress|Psychological   
5781  Emotional Problem|Anxiety Disorder of Childhoo...   
5782                                       Chronic Pain   

                                          Interventions  \
5778  Behavioral: A-CHESS self-monitored|Behavioral:...   
5779  Other: Antibiotic treatment|Other: No antibiot...   
5780                                   Behavioral: Yoga   
5781                                                NaN   
5782  Behavioral: Brief Cognitive Behavioral Therapy...   

                                       Outcome Measures  \
5778  Number of risky drinking days|Number of patien...   
5779  Antibiotic treatment rates in hospitalized chi...   
5780  Perceived Stress|Task switching paradigm|Digit...   
5781  The child's impact of mental health problems r...   
5782  Defense and Veterans Pain Rating Scale (DVPRS)...   

                                  Sponsor/Collaborators Gender  \
5778  University of Wisconsin, Madison|National Inst...    All   
5779  ARCIM Institute Academic Research in Complemen...    All   
5780         University of Illinois at Urbana-Champaign    All   
5781  Mental Health Services in the Capital Region, ...    All   
5782  The University of Texas Health Science Center ...    All   

                                               Age          Phases  \
5778  21 Years to 100 Years   (Adult, Older Adult)  Not Applicable   
5779         3 Months to 18 Years   (Child, Adult)             NaN   
5780                18 Years to 64 Years   (Adult)  Not Applicable   
5781          8 Years to 20 Years   (Child, Adult)             NaN   
5782     18 Years and older   (Adult, Older Adult)  Not Applicable   

      Enrollment      Funded Bys      Study Type  \
5778       566.0       Other|NIH  Interventional   
5779       200.0           Other   Observational   
5780        88.0           Other  Interventional   
5781       396.0           Other   Observational   
5782       716.0  Other|U.S. Fed  Interventional   

                                          Study Designs  \
5778  Allocation: Randomized|Intervention Model: Par...   
5779  Observational Model: Cohort|Time Perspective: ...   
5780  Allocation: Randomized|Intervention Model: Par...   
5781  Observational Model: Other|Time Perspective: P...   
5782  Allocation: Randomized|Intervention Model: Par...   

                                              Other IDs         Start Date  \
5778  2019-0337|R01AA024150|A532007|SMPH/FAMILY MED/...     March 23, 2020   
5779                                             PKA-03         April 2021   
5780                                              21584  February 10, 2021   
5781                         MHSCRDenmark, F-61502-03-1     March 22, 2021   
5782                                       HSC20200520H      February 2021   

     Primary Completion Date    Completion Date       First Posted  \
5778             August 2022         April 2023       July 8, 

In [7]:
# First 15 rows of the dataset
df.head(15)

Rank   NCT Number                                              Title  \
0      1  NCT04785898  Diagnostic Performance of the ID Now™ COVID-19...   
1      2  NCT04595136  Study to Evaluate the Efficacy of COVID19-0001...   
2      3  NCT04395482  Lung CT Scan Analysis of SARS-CoV2 Induced Lun...   
3      4  NCT04416061  The Role of a Private Hospital in Hong Kong Am...   
4      5  NCT04395924         Maternal-foetal Transmission of SARS-Cov-2   
5      6  NCT04516954          Convalescent Plasma for COVID-19 Patients   
6      7  NCT04476940  COVID-19 Breastfeeding Guideline for African-A...   
7      8  NCT04634214  The Severity of COVID 19 in Diabetes and Non-d...   
8      9  NCT04602884  Early Detection of COVID-19 Using Breath Analysis   
9     10  NCT04384588  COVID19-Convalescent Plasma for Treating Patie...   
10    11  NCT04355897  CoVID-19 Plasma in Treatment of COVID-19 Patients   
11    12  NCT04412265          Frailty in Elderly Patients With COVID-19   
12    13  NCT04659759  COVID-19 Pregnancy Related Immunological, Clin...   
13    14  NCT04427332     Smell and Taste Disorders in COVID-19 Patients   
14    15  NCT04842708  Evaluation of Anti-COVID 19 Pfizer Vaccination...   

         Acronym                   Status         Study Results  \
0    COVID-IDNow   Active, not recruiting  No Results Available   
1       COVID-19       Not yet recruiting  No Results Available   
2    TAC-COVID19               Recruiting  No Results Available   
3       COVID-19   Active, not recruiting  No Results Available   
4   TMF-COVID-19               Recruiting  No Results Available   
5           CPCP  Enrolling by invitation  No Results Available   
6       COVID-BF       Not yet recruiting  No Results Available   
7        COVID19       Not yet recruiting  No Results Available   
8       COVID-19                Suspended  No Results Available   
9     FALP-COVID               Recruiting  No Results Available   
10           NaN               Recruiting  No Results Available   
11     FRA-COVID               Recruiting  No Results Available   
12   COVID-PRICE               Recruiting  No Results Available   
13  COVID-19 ORL                Completed  No Results Available   
14      COVID-19               Recruiting  No Results Available   

                                           Conditions  \
0                                             Covid19   
1                                SARS-CoV-2 Infection   
2                                             covid19   
3                                               COVID   
4   Maternal Fetal Infection Transmission|COVID-19...   
5                                            COVID 19   
6                     Covid19|Exclusive Breastfeeding   
7                              Covid19|Type2 Diabetes   
8                                             Covid19   
9   COVID-19 Infection|Cancer Patients|General Pop...   
10                                           COVID 19   
11                                            Covid19   
12                                            Covid19   
13                                            covid19   
14                                            Covid19   

                                        Interventions  \
0    Diagnostic Test: ID Now™ COVID-19 Screening Test   
1     Drug: Drug COVID19-0001-USR|Drug: normal saline   
2   Other: Lung CT scan analysis in COVID-19 patients   
3           Diagnostic Test: COVID 19 Diagnostic Test   
4   Diagnostic Test: Diagnosis of SARS-Cov2 by RT-...   
5            Biological: Convalescent COVID 19 Plasma   
6          Behavioral: COVID-19 Breastfeeding Support   
7                                                 NaN   
8   Diagnostic Test: Breath biopsy sampling using ...   
9   Biological: Convalescent Plasma from COVID-19 ...   
10           Biological: Convalescent COVID 19 Plasma   
11  Other: Relation between frailty and clinical o...   
12  Other: COVID-19 exposure|Biological: COVID-19 ...   
13 

In [ ]:
# Last 15 rows of the dataset
df.tail(15)

Rank   NCT Number                                              Title  \
5768  5769  NCT04734795  The Prevalence of Dysfunctional Breathing in C...   
5769  5770  NCT04190368  Team Clinic: Virtual Expansion of an Innovativ...   
5770  5771  NCT03392883  Scaling Up Science-based Mental Health Interve...   
5771  5772  NCT04301518  Prematurity Risk Assessment Combined With Clin...   
5772  5773  NCT04607902  Harnessing Network Science to Personalize Scal...   
5773  5774  NCT04639661  Predictors of Periodontal Outcomes Post-sanati...   
5774  5775  NCT04180709  CBT to Reduce Insomnia and Improve Social Reco...   
5775  5776  NCT04335643  Telehealth CBT for Adolescents and Young Adult...   
5776  5777  NCT04589377  Mindfulness to Mitigate Psychological Threat a...   
5777  5778  NCT04574466  Scaling-up Psychological Interventions With Sy...   
5778  5779  NCT04011644  Mobile Health for Alcohol Use Disorders in Cli...   
5779  5780  NCT04681339  Antibiotic Prescription in Children Hospitaliz...   
5780  5781  NCT04740229  Moderate-intensity Flow-based Yoga Effects on ...   
5781  5782  NCT04804917           3-year Follow-up of the Mind My Mind RCT   
5782  5783  NCT04680000  Chronic Pain Management In Primary Care Using ...   

           Acronym                   Status         Study Results  \
5768           NaN               Recruiting  No Results Available   
5769           NaN       Not yet recruiting  No Results Available   
5770         DIADA   Active, not recruiting  No Results Available   
5771         PRIME               Recruiting  No Results Available   
5772           NaN               Recruiting  No Results Available   
5773           NaN  Enrolling by invitation  No Results Available   
5774         CRISP               Recruiting  No Results Available   
5775          cSLE               Recruiting  No Results Available   
5776           NaN               Recruiting  No Results Available   
5777           NaN               Recruiting  No Results Available   
5778           NaN               Recruiting  No Results Available   
5779           NaN       Not yet recruiting  No Results Available   
5780           NaN               Recruiting  No Results Available   
5781  MindMyMindFU               Recruiting  No Results Available   
5782           NaN       Not yet recruiting  No Results Available   

                                             Conditions  \
5768  Dysfunctional Breathing|Asthma in Children|Asthma   
5769                                    Type 1 Diabetes   
5770                 Depression|Problematic Alcohol Use   
5771                        Preterm Labor|Preterm Birth   
5772                                         Depression   
5773            Periodontal Diseases|Periodontal Pocket   
5774                Psychotic Disorders|Psychosis|Sleep   
5775                       Systemic Lupus Erythematosus   
5776                                             Stress   
5777  Distress|PTSD|Anxiety|Depression|Trauma|Functi...   
5778                      Alcohol Drinking|Telemedicine   
5779  Community Acquired Pneumonia in Children|Antib...   
5780                               Stress|Psychological   
5781  Emotional Problem|Anxiety Disorder of Childhoo...   
5782                                       Chronic Pain   

                                          Interventions  \
5768                                                NaN   
5769       Other: Team Clinic Care|Other: Standard Care   
5770                                  Behavioral: Laddr   
5771            Other: Multimodal intervention strategy   
5772  Behavioral: Supportive Therapy SSI|Behavioral:...   
5773                                                NaN   
5774                                    Device: Sleepio   
5775                                  Behavioral: TEACH   
5776                   Behavioral: Mindfulness Training   
5777                Behavioral: Problem Management Plus   
5778  Behavioral: A-CHESS self-monitored|Behavioral:...   

In [8]:
# First 30 rows of the dataset
df.head(30)

Rank   NCT Number                                              Title  \
0      1  NCT04785898  Diagnostic Performance of the ID Now™ COVID-19...   
1      2  NCT04595136  Study to Evaluate the Efficacy of COVID19-0001...   
2      3  NCT04395482  Lung CT Scan Analysis of SARS-CoV2 Induced Lun...   
3      4  NCT04416061  The Role of a Private Hospital in Hong Kong Am...   
4      5  NCT04395924         Maternal-foetal Transmission of SARS-Cov-2   
5      6  NCT04516954          Convalescent Plasma for COVID-19 Patients   
6      7  NCT04476940  COVID-19 Breastfeeding Guideline for African-A...   
7      8  NCT04634214  The Severity of COVID 19 in Diabetes and Non-d...   
8      9  NCT04602884  Early Detection of COVID-19 Using Breath Analysis   
9     10  NCT04384588  COVID19-Convalescent Plasma for Treating Patie...   
10    11  NCT04355897  CoVID-19 Plasma in Treatment of COVID-19 Patients   
11    12  NCT04412265          Frailty in Elderly Patients With COVID-19   
12    13  NCT04659759  COVID-19 Pregnancy Related Immunological, Clin...   
13    14  NCT04427332     Smell and Taste Disorders in COVID-19 Patients   
14    15  NCT04842708  Evaluation of Anti-COVID 19 Pfizer Vaccination...   
15    16  NCT04367805  COVID-19 Infection in Patients With Hepatocell...   
16    17  NCT04609969  Diagnostic Performance Evaluation of a Novel S...   
17    18  NCT04817553  Impact of COVID-19 on the Clinical Outcomes an...   
18    19  NCT04632355  Musculoskeletal Pain in Patients With Covid-19...   
19    20  NCT04407585  Testing the Accuracy of a Digital Test to Diag...   
20    21  NCT04657510                    Femoral frACturEs and COVID-19.   
21    22  NCT04473170  Study Evaluating the Safety and Efficacy of Au...   
22    23  NCT04497298  Clinical Trial to Evaluate the Safety and Immu...   
23    24  NCT04549831     Genetic Bases of COVID-19 Clinical Variability   
24    25  NCT04706390  Comparison of the Immune Response to Natural C...   
25    26  NCT04632719  The MentalPlus® for Assessment and Rehabilitat...   
26    27  NCT04427345               Predictive Factors COVID-19 Patients   
27    28  NCT04424004  MURDOCK Cabarrus County COVID-19 Prevalence an...   
28    29  NCT04407494  Diagnostic Value of Patient - Reported and Cli...   
29    30  NCT04360811  Assessment of Obstetric, Fetal and Neonatal Ri...   

          Acronym                   Status         Study Results  \
0     COVID-IDNow   Active, not recruiting  No Results Available   
1        COVID-19       Not yet recruiting  No Results Available   
2     TAC-COVID19               Recruiting  No Results Available   
3        COVID-19   Active, not recruiting  No Results Available   
4    TMF-COVID-19               Recruiting  No Results Available   
5            CPCP  Enrolling by invitation  No Results Available   
6        COVID-BF       Not yet recruiting  No Results Available   
7         COVID19       Not yet recruiting  No Results Available   
8        COVID-19                Suspended  No Results Available   
9      FALP-COVID               Recruiting  No Results Available   
10            NaN               Recruiting  No Results Available   
11      FRA-COVID               Recruiting  No Results Available   
12    COVID-PRICE               Recruiting  No Results Available   
13   COVID-19 ORL                Completed  No Results Available   
14       COVID-19               Recruiting  No Results Available   
15  COVID19-CHIEF               Recruiting  No Results Available   
16     COVID-VIRO                Completed  No Results Available   
17     IgG4-COVID               Recruiting  No Results Available   
18   Covid19-Pain       Not yet recruiting  No Results Available   
19            NaN               Recruiting  No Results Available   
20  FACE COVID-19               Recruiting  No Results Available   
21   SENTAD-COVID                Completed  No Results Available   
22   COVID-19-101   Active, not recruiting  No Results Available   
23      GEN-COV

In [ ]:
# Last 30 rows of the dataset
df.tail(30)

Rank   NCT Number                                              Title  \
5753  5754  NCT04646291  Retrospective Study on the Use, Efficiency, an...   
5754  5755  NCT04735406                        The MS-LINK™ Outcomes Study   
5755  5756  NCT02495753  Vaginal Cleansing Before Cesarean Delivery to ...   
5756  5757  NCT03465280  Airway Intervention Registry (AIR): Recurrent ...   
5757  5758  NCT04576377  Dynamics of the Immune Responses to Repeat Inf...   
5758  5759  NCT04713735  Impact of Lactoferrin, a Dietary Supplement, v...   
5759  5760  NCT03873467  Group Lifestyle Balance™ for Individuals With ...   
5760  5761  NCT03594734  Group Lifestyle Balance™ for Individuals With ...   
5761  5762  NCT03728257  Lung Transplant G0 (LTGO): Improving Self-Mana...   
5762  5763  NCT04782518       Personalizing Exercise for Parkinson Disease   
5763  5764  NCT04783454  Preventing Neck & Back Pain in Teleworking Off...   
5764  5765  NCT03823469  Evaluating the Impact of a Culinary Coaching T...   
5765  5766  NCT04429061  Reaching 90 90 90 in Adolescents in Zambia: Us...   
5766  5767  NCT04184895  A Study to Assess the Safety, Tolerability and...   
5767  5768  NCT03922334                        Navigating New Motherhood 2   
5768  5769  NCT04734795  The Prevalence of Dysfunctional Breathing in C...   
5769  5770  NCT04190368  Team Clinic: Virtual Expansion of an Innovativ...   
5770  5771  NCT03392883  Scaling Up Science-based Mental Health Interve...   
5771  5772  NCT04301518  Prematurity Risk Assessment Combined With Clin...   
5772  5773  NCT04607902  Harnessing Network Science to Personalize Scal...   
5773  5774  NCT04639661  Predictors of Periodontal Outcomes Post-sanati...   
5774  5775  NCT04180709  CBT to Reduce Insomnia and Improve Social Reco...   
5775  5776  NCT04335643  Telehealth CBT for Adolescents and Young Adult...   
5776  5777  NCT04589377  Mindfulness to Mitigate Psychological Threat a...   
5777  5778  NCT04574466  Scaling-up Psychological Interventions With Sy...   
5778  5779  NCT04011644  Mobile Health for Alcohol Use Disorders in Cli...   
5779  5780  NCT04681339  Antibiotic Prescription in Children Hospitaliz...   
5780  5781  NCT04740229  Moderate-intensity Flow-based Yoga Effects on ...   
5781  5782  NCT04804917           3-year Follow-up of the Mind My Mind RCT   
5782  5783  NCT04680000  Chronic Pain Management In Primary Care Using ...   

           Acronym                   Status         Study Results  \
5753           NaN                Completed  No Results Available   
5754           NaN       Not yet recruiting  No Results Available   
5755           NaN   Active, not recruiting  No Results Available   
5756       AIR:RRP               Recruiting  No Results Available   
5757         DRIVE               Recruiting  No Results Available   
5758           NaN               Recruiting  No Results Available   
5759       GLB-CVA   Active, not recruiting  No Results Available   
5760           NaN   Active, not recruiting  No Results Available   
5761          LTGO               Recruiting  No Results Available   
5762           PEP               Recruiting  No Results Available   
5763  Prevent@HOME               Recruiting  No Results Available   
5764           NaN               Recruiting  No Results Available   
5765        SKILLZ               Recruiting  No Results Available   
5766           NaN               Recruiting  No Results Available   
5767          NNM2               Recruiting  No Results Available   
5768           NaN               Recruiting  No Results Available   
5769           NaN       Not yet recruiting  No Results Available   
5770         DIADA   Active, not recruiting  No Results Available   
5771         PRIME               Recruiting  No Results Available   
5772           NaN               Recruiting  No Results Available   
5773           NaN  Enrolling by invitation  No Results Available   
5774         CRISP               Recruiting  No Results Available   


### 2. Extract and document key insights by exploring column data types and applying info and describe

In [9]:
# Data types of the columns
df.dtypes

Rank                         int64
NCT Number                  object
Title                       object
Acronym                     object
Status                      object
Study Results               object
Conditions                  object
Interventions               object
Outcome Measures            object
Sponsor/Collaborators       object
Gender                      object
Age                         object
Phases                      object
Enrollment                 float64
Funded Bys                  object
Study Type                  object
Study Designs               object
Other IDs                   object
Start Date                  object
Primary Completion Date     object
Completion Date             object
First Posted                object
Results First Posted        object
Last Update Posted          object
Locations                   object
Study Documents             object
URL                         object
dtype: object

In [10]:
# Count of data types
df.dtypes.value_counts()

object     25
int64       1
float64     1
dtype: int64

*There is **1** integer, **1** float and **25** object data types.*

In [11]:
# Overall information about the dataset
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 5783 entries, 0 to 5782
Data columns (total 27 columns):
 #   Column                   Non-Null Count  Dtype  
---  ------                   --------------  -----  
 0   Rank                     5783 non-null   int64  
 1   NCT Number               5783 non-null   object 
 2   Title                    5783 non-null   object 
 3   Acronym                  2480 non-null   object 
 4   Status                   5783 non-null   object 
 5   Study Results            5783 non-null   object 
 6   Conditions               5783 non-null   object 
 7   Interventions            4897 non-null   object 
 8   Outcome Measures         5748 non-null   object 
 9   Sponsor/Collaborators    5783 non-null   object 
 10  Gender                   5773 non-null   object 
 11  Age                      5783 non-null   object 
 12  Phases                   3322 non-null   object 
 13  Enrollment               5749 non-null   float64
 14  Funded Bys              


The dataset contains **5783 entries** and **27 columns**. <br>
Among the columns, there is:<br> **1 integer** type,<br> **1 float** type,<br> and **25 object** types.<br> Several columns, namely Acronym, Interventions, Outcome Measures, Gender, Phases, Enrollment, Study Designs, Other IDs, Start Date, Primary Completion Date, Completion Date, Results First Posted, Locations, and Study Documents, have missing values. The dataset occupies over 1.2 MB of memory.

In [12]:
# Descriptive statistics about the dataset (numeric columns)
df.describe()

Rank    Enrollment
count  5783.000000  5.749000e+03
mean   2892.000000  1.831949e+04
std    1669.552635  4.045437e+05
min       1.000000  0.000000e+00
25%    1446.500000  6.000000e+01
50%    2892.000000  1.700000e+02
75%    4337.500000  5.600000e+02
max    5783.000000  2.000000e+07

**Missing Values:**

The Rank column has no missing values.
There are **34** missing values in the Enrollment column.
Rank Statistics:

Minimum Rank: **1** <br>
Maximum Rank: **5783**<br>

**Enrollment Statistics:**
<br>
Minimum Enrolled Patients: **0**<br>
Maximum Enrolled Patients: **20,000,000**<br>
Average Enrolled Patients: **18,319.49**

In [ ]:
# Descriptive statistics about object data types
df.describe(include='object')

NCT Number                                              Title  \
count          5783                                               5783   
unique         5783                                               5775   
top     NCT04785898  Study Assessing Vagus Nerve Stimulation in CoV...   
freq              1                                                  2   

         Acronym      Status         Study Results Conditions  \
count       2480        5783                  5783       5783   
unique      2338          12                     2       3067   
top     COVID-19  Recruiting  No Results Available   COVID-19   
freq          47        2805                  5747        720   

                 Interventions Outcome Measures  \
count                     4897             5748   
unique                    4337             5687   
top     Other: No intervention        Mortality   
freq                        32                5   

                          Sponsor/Collaborators Gender  \
count                                      5783   5773   
unique                                     3631      3   
top     Assistance Publique - Hôpitaux de Paris    All   
freq                                         78   5567   

                                              Age          Phases Funded Bys  \
count                                        5783            3322       5783   
unique                                        417               8         26   
top     18 Years and older   (Adult, Older Adult)  Not Applicable      Other   
freq                                         2885            1354       4488   

            Study Type                                      Study Designs  \
count             5783                                               5748   
unique               9                                                267   
top     Interventional  Observational Model: Cohort|Time Perspective: ...   
freq              3322                                               1011   

       Other IDs   Start Date Primary Completion Date    Completion Date  \
count       5782         5749                    5747               5747   
unique      5734          654                     877                978   
top     COVID-19  May 1, 2020       December 31, 2020  December 31, 2021   
freq           6          113                     122                179   

          First Posted Results First Posted Last Update Posted  \
count             5783                   36               5783   
unique             438                   33                269   
top     April 24, 2020     November 4, 2020      April 8, 2021   
freq               108                    2                109   

                                 Locations  \
count                                 5198   
unique                                4255   
top     Uhmontpellier, Montpellier, France   
freq                                    19   

                                          Study Documents  \
count                                                 182   
unique                                                182   
top     "Statistical Analysis Plan", https://ClinicalT...   
freq                                                    1   

                                                URL  
count                                          5783  
unique                                         5783  
top     https://ClinicalTrials.gov/show/NCT04785898  
freq                                              1


**Unique Observations (NCT Number): 5783**

**Unique Values:**
```
Title: 5775<br>
Acronym: 2338<br>
Status: 12<br>
Study Results: 2<br>
Conditions: 3067<br>
Interventions: 4337<br>
Outcome Measures: 5687<br>
Sponsors/Collaborators: 3631<br>
Genders: 3<br>
Age: 417<br>
Phases: 8<br>
Funded Bys: 26<br>
Study Types: 9<br>
Study Designs: 267<br>
Other IDs: 5734<br>
Start Dates: 654<br>
Primary Completion Dates: 877<br>
Completion Dates: 978<br>
First Posted: 438<br>
Results First Posted: 33<br>
Last Update Posted: 269<br>
Locations: 4255<br>
Study Documents: 182<br>

```
**Top Occurrences:**<br>


```
Top Acronym: COVID-19 (Occurrences: 47)<br>
Top Status: Recruiting (Occurrences: 2805)<br>
Top Study Results: No Results Available (Occurrences: 5747)<br>
Top Conditions: COVID-19 (Occurrences: 720)<br>
Top Outcome Measures: Mortality (Occurrences: 5)<br>
Top Sponsor/Collaborators: Assistance Publique - Hôpitaux de Paris (Occurrences: 78)<br>
Top Gender: All (Occurrences: 5567)<br>
Top Age: 18 Years and older (Adult, Older Adult) (Occurrences: 2885)<br>
Top Phases: Not Applicable (Occurrences: 1354)<br>
Top Funded Bys: Other (Occurrences: 4488)<br>
Top Study Types: Interventional (Occurrences: 3322)<br>
Top Study Designs: Observational Model: Cohort|Time Perspective: ... (Occurrences: 1011)<br>
Top Start Date: May 1, 2020 (Occurrences: 113)<br>
Top Primary Completion Date: December 31, 2020 (Occurrences: 122)<br>
Top Completion Date: December 31, 2021 (Occurrences: 179)<br>
Top First Posted Date: April 24, 2020 (Occurrences: 108)<br>
Top Results First Posted Date: November 4, 2020 (Occurrences: 2)<br>
Top Last Update Posted Date: April 8, 2021 (Occurrences: 109)<br>
Top Locations: Uhmontpellier, Montpellier, France (Occurrences: 19)<br>

```



### Key Insights



```
1. There are **5783** clinical trials in total.
2. Minimum enrolled patients for a clinical trial are **0**.
3. Maximum enrolled patients for a clinical trial are **20000000**.
4. Average enrolled patients for a clinical trial are **18319.49**.
5. **5747** clinical trials have **NO** study results available.
6. Largest number of clinical trials have been sponsored by **Assistance Publique - Hôpitaux de Paris**.
7. Largest number of clinical trials have been done for **All** genders.
8. Almost half of the clinical trials (**2885**) have been done for age **18 Years and older (Adult, Older Adult)**.
9. More clinical trials have been done for **Interventional** study type (**3322** in total) than Observational study type.
```



### 3. Determine the missing values in the whole dataset and analyze missing values in each column

In [13]:
# Total missing values
df.isnull().sum().sum()

18804

*There are total **18804** missing values in the whole dataset.*

In [14]:
# Missing values in each column
df.isnull().sum()

Rank                          0
NCT Number                    0
Title                         0
Acronym                    3303
Status                        0
Study Results                 0
Conditions                    0
Interventions               886
Outcome Measures             35
Sponsor/Collaborators         0
Gender                       10
Age                           0
Phases                     2461
Enrollment                   34
Funded Bys                    0
Study Type                    0
Study Designs                35
Other IDs                     1
Start Date                   34
Primary Completion Date      36
Completion Date              36
First Posted                  0
Results First Posted       5747
Last Update Posted            0
Locations                   585
Study Documents            5601
URL                           0
dtype: int64

*Following columns have missing values:*


```
* *Acronym (**3303** missing values)*
* *Interventions (**886** missing values)*
* *Outcome Measures (**35** missing values)*
* *Gender (**10** mising values)*
* *Phases (**2461** missing values)*
* *Enrollment (**34** missing values)*
* *Study Designs (**35** missing values)*
* *Other IDs (**1** missing value)*
* *Start Date (**34** missing values)*
* *Primary Completion Date (**36** missing values)*
* *Completion Date (**36** missing values)*
* *Results First Posted (**5747** missing values)*
* *Locations (**585** missing values)*
* *Study Documents (**5601** missing values)*
```




### 4. Calculate the sum of duplicate rows

In [ ]:
# Number of duplicate rows
df.duplicated().sum()

0

*There are **NO** duplicate rows in the dataset.*

### 5. Make 5 questions and solve them by using group by

#### Q1. Total number of enrollments by phases?

In [ ]:
# Enrollments by phases
df.groupby('Phases')[['Enrollment']].sum()

Enrollment
Phases                     
Early Phase 1        3496.0
Not Applicable   24784554.0
Phase 1             23559.0
Phase 1|Phase 2     46799.0
Phase 2            350044.0
Phase 2|Phase 3    303259.0
Phase 3           1466515.0
Phase 4            244453.0

#### Q2. Average enrollments by gender?

In [ ]:
# Average enrollments by gender
df.groupby('Gender')[['Enrollment']].mean()

Enrollment
Gender              
All     18984.795122
Female   1365.456790
Male      260.886364

#### Q3: Total numbers of enrollments for each status type?   

In [15]:
# Enrollments for each individual status
df.groupby('Status')[['Enrollment']].sum()

Enrollment
Status                               
Active, not recruiting     31956666.0
Approved for marketing            0.0
Available                         0.0
Completed                  28522364.0
Enrolling by invitation     2591852.0
No longer available               0.0
Not yet recruiting         11625671.0
Recruiting                 30558922.0
Suspended                     57732.0
Temporarily not available         0.0
Terminated                     5533.0
Withdrawn                         0.0

#### Q4. Total number of enrollments by study results?

In [ ]:
# Total enrollments by study results
df.groupby('Study Results')[['Enrollment']].sum()

Enrollment
Study Results                    
Has Results               19123.0
No Results Available  105299617.0

#### Q5. Maximum enrollments by study type?

In [ ]:
# Maximum enrollments by study type
df.groupby('Study Type')[['Enrollment']].max()

Enrollment
Study Type                                                    
Expanded Access                                            NaN
Expanded Access:Individual Patients                        NaN
Expanded Access:Individual Patients|Intermediat...         NaN
Expanded Access:Individual Patients|Treatment I...         NaN
Expanded Access:Intermediate-size Population               NaN
Expanded Access:Intermediate-size Population|Tr...         NaN
Expanded Access:Treatment IND/Protocol                     NaN
Interventional                                      20000000.0
Observational                                       12000000.0

### 6. Make 5 questions and solve them by using conditional statements

#### Q1. Display the clinical trials with Acronym 'COVID-19'.

In [ ]:
df[df['Acronym'] == 'COVID-19']

Rank   NCT Number                                              Title  \
1        2  NCT04595136  Study to Evaluate the Efficacy of COVID19-0001...   
3        4  NCT04416061  The Role of a Private Hospital in Hong Kong Am...   
8        9  NCT04602884  Early Detection of COVID-19 Using Breath Analysis   
14      15  NCT04842708  Evaluation of Anti-COVID 19 Pfizer Vaccination...   
33      34  NCT04602871  Early Detection of COVID-19 Using Breath Analy...   
54      55  NCT04508075  Efficacy, Safety and Immunogenicity Study of S...   
57      58  NCT04602949  Detection of COVID-19 Using Breath Analysis - ...   
67      68  NCT04668404  Study of Coagulation Profile and Role of Hepar...   
69      70  NCT04339322  Characteristics and Outcome of Coronavirus Dis...   
74      75  NCT04323787  Viral Infection and Respiratory Illness Univer...   
75      76  NCT04510207  A Study to Evaluate The Efficacy, Safety and I...   
184    185  NCT04483622  Immunoglobulin G Antibody Immune Response Prof...   
203    204  NCT04474483  Safety and Efficacy of Melatonin in Outpatient...   
219    220  NCT04376788  Exchange Transfusion Versus Plasma From Conval...   
273    274  NCT04763395      Liver Function Test Abnormalities in COVID-19   
319    320  NCT04416139  Mesenchymal Stem Cell for Acute Respiratory Di...   
320    321  NCT04380532              Tableted COVID-19 Therapeutic Vaccine   
329    330  NCT04535791  Efficacy of Vitamin D Supplementation to Preve...   
358    359  NCT04818281  Study of a Severe Acute Respiratory Syndrome C...   
371    372  NCT04839913  Seroprevalence of SARS-CoV-2 in Unselected Sur...   
411    412  NCT04336462  Hydrogen-Oxygen Generator With Nebulizer in th...   
432    433  NCT04515199  Home Management of Adult Egyptian Mild COVID-1...   
494    495  NCT04764773  Persistence of Symptoms After Improvement of A...   
509    510  NCT04723446  Efficacy of Mouthwashes in Reducing Severe Acu...   
545    546  NCT04695652  A Study to Evaluate MVC-COV1901 Vaccine Agains...   
582    583  NCT04344457  Evaluate the Efficacy and Safety of Oral Hydro...   
620    621  NCT04815018  Shifts in the Respiratory Microbiome and Clini...   
625    626  NCT04828915  Early Detection of Clinical Deterioration in P...   
689    690  NCT04766294  Macrophage Activation Markers in COVID-19 Pati...   
752    753  NCT04609774  Aerosol Transmission of Severe Acute Respirato...   
765    766  NCT04491994  Clearing the Fog: Is Hydroxychloroquine Effect...   
784    785  NCT04322682  Colchicine Coronavirus SARS-CoV2 Trial (COLCOR...   
813    814  NCT04828564  Efficacy and Safety of Favipiravir and Ribavir...   
932    933  NCT04388631  Detection Rate of SARS-CoV-2 in Male Genitouri...   
1157  1158  NCT04685213  Electrical Stimulation for Critically Ill Covi...   
1172  1173  NCT04490486  Umbilical Cord Tissue (UC) Derived Mesenchymal...   
1303  1304  NCT04502667  Efficacy of Vitamin D Treatment in Pediatric P...   
1599  1600  NCT04256395  Efficacy of a Self-test and Self-alert Mobile ...   
1644  1645  NCT04613245  Health-Related Quality of Life of Patients Wit...   
2002  2003  NCT04757285  Copeptin and Psychological Stress of Medic Dur...   
2013  2014  NCT04522986  An Exploratory Study of ADR-001 in Patients Wi...   
2033  2034  NCT04331886  An Observational Study of Patients With Corona...   
2302  2303  NCT04400305  "Increasing Physical Activity in Canadian Adul...   
3487  3488  NCT04738435  Safety of the Sputnik V Vaccine in Health Pers...   
3724  3725  NCT04414618  A Study of Opaganib in Coronavirus Disease 201...   
3811  3812  NCT04346927  Telerehabilitation for Patients Diagnosed With...   
3969  3970  NCT04319211  Effects of Social Isolation From Coronavirus; ...   

       Acronym                  Status         Study Results  \
1     COVID-19      Not yet recruiting  No Results Available   
3     COVID-19  Active, not recruiting  No Results Available   
8     COVID-19               Suspended  No Results Available   
14   

#### Q2. Display the clinical trials with 'Rank' from 10 to 20.

In [16]:
df[(df['Rank'] >= 10) & (df['Rank'] <= 20)]

Rank   NCT Number                                              Title  \
9     10  NCT04384588  COVID19-Convalescent Plasma for Treating Patie...   
10    11  NCT04355897  CoVID-19 Plasma in Treatment of COVID-19 Patients   
11    12  NCT04412265          Frailty in Elderly Patients With COVID-19   
12    13  NCT04659759  COVID-19 Pregnancy Related Immunological, Clin...   
13    14  NCT04427332     Smell and Taste Disorders in COVID-19 Patients   
14    15  NCT04842708  Evaluation of Anti-COVID 19 Pfizer Vaccination...   
15    16  NCT04367805  COVID-19 Infection in Patients With Hepatocell...   
16    17  NCT04609969  Diagnostic Performance Evaluation of a Novel S...   
17    18  NCT04817553  Impact of COVID-19 on the Clinical Outcomes an...   
18    19  NCT04632355  Musculoskeletal Pain in Patients With Covid-19...   
19    20  NCT04407585  Testing the Accuracy of a Digital Test to Diag...   

          Acronym              Status         Study Results  \
9      FALP-COVID          Recruiting  No Results Available   
10            NaN          Recruiting  No Results Available   
11      FRA-COVID          Recruiting  No Results Available   
12    COVID-PRICE          Recruiting  No Results Available   
13   COVID-19 ORL           Completed  No Results Available   
14       COVID-19          Recruiting  No Results Available   
15  COVID19-CHIEF          Recruiting  No Results Available   
16     COVID-VIRO           Completed  No Results Available   
17     IgG4-COVID          Recruiting  No Results Available   
18   Covid19-Pain  Not yet recruiting  No Results Available   
19            NaN          Recruiting  No Results Available   

                                           Conditions  \
9   COVID-19 Infection|Cancer Patients|General Pop...   
10                                           COVID 19   
11                                            Covid19   
12                                            Covid19   
13                                            covid19   
14                                            Covid19   
15                  Hepatocellular Carcinoma|COVID-19   
16                       Covid19|SARS-CoV-2 Infection   
17                       IgG4 Related Disease|Covid19   
18                                            Covid19   
19                                           Covid-19   

                                        Interventions  \
9   Biological: Convalescent Plasma from COVID-19 ...   
10           Biological: Convalescent COVID 19 Plasma   
11  Other: Relation between frailty and clinical o...   
12  Other: COVID-19 exposure|Biological: COVID-19 ...   
13  Other: Investigation of smell and taste disorders   
14       Diagnostic Test: vaccination against COVID19   
15    Diagnostic Test: nasopharyngeal Covid 19 RT-PCR   
16  Diagnostic Test: RT-qPCR test|Diagnostic Test:...   
17                         Other: exposure to COVID19   
18                                   Other: Interview   
19            Diagnostic Test: Covid-19 swab PCR test   

                                     Outcome Measures  \
9   in-hospital mortality secondary to COVID-19 am...   
10  Reduce mortality|Reduce requirement for mechan...   
11  Development of a tool to measure frailty|A "pr...   
12  Maternal COVID-19 serology (IgG and IgM)|Mater...   
13  Identification of demographic and clinical fac...   
14  Association between breath VOCs and IgG in blo...   
15  Incidence of COVID-19 infection in patients wi...   
16  Evaluation of COVID VIRO® diagnostic specifici...   
17  Incidence of COVID-19 in IgG4 patients with pa...   
18  Self-reported Version of the Leeds Assessment ...   
19                               SARS-CoV-2 infection   

                                Sponsor/Collaborators  Gender  \
9   Fundacion Arturo Lopez Perez|Confederación de ...     All   
10                                The Christ Hospital     All   
11                       University of Milano Bicocca     All   
12                T

#### Q3. Display all the clinical trials where 'Start Date' is March 1, 2021.

In [17]:
df[df['Start Date'] == 'March 1, 2021']

Rank   NCT Number                                              Title  \
117    118  NCT04834752  Effect of H2 Receptor Antagonist and Proton Pu...   
176    177  NCT04839068  the Impact of Covid-19 Pandemic Stress on Preg...   
278    279  NCT04832932                  The COVID-19 Back-to-Normal Study   
766    767  NCT04826640  Observational Maternal COVID-19 Vaccination Study   
1413  1414  NCT04836754  The Effect of Covid-19 on Breast Milk Microbio...   
1620  1621  NCT04825093  Vitamin D Supplementarion in Pregnant Women at...   
1638  1639  NCT04808882        ANTIcoagulation in Severe COVID-19 Patients   
2376  2377  NCT04844346  Plant Stanol Esters and COVID-19 Vaccination R...   
2439  2440  NCT04565392  Remotely-conducted Trial of Famotidine vs Plac...   
2592  2593  NCT04830852  Pediatric SARS-CoV-2 and MIS-C Long-term Follo...   
2822  2823  NCT04843787  A Randomized, Double Blind, Placebo-Controlled...   
3066  3067  NCT04813718       Post COVID-19 Syndrome and the Gut-lung Axis   
3447  3448  NCT04729595  Study to Evaluate Tempol (MBM-02) to Help Prev...   
3471  3472  NCT04807075  A Wearable, Battery-free Screening System for ...   
3785  3786  NCT04751630  Effects of COVID-19 Hospitalization on Physica...   
4469  4470  NCT04604743  Clinic-based HPV and COVID-19 Vaccine Promotin...   
5457  5458  NCT04742426  Personal Protective Equipment Affect Team Perf...   
5523  5524  NCT04832152  Telehealth Intervention for Autism Spectrum Di...   
5651  5652  NCT04838288  Cognitive Outcomes in Stable Renal Transplant ...   

           Acronym              Status         Study Results  \
117            NaN  Not yet recruiting  No Results Available   
176            NaN  Not yet recruiting  No Results Available   
278            NaN  Not yet recruiting  No Results Available   
766            NaN  Not yet recruiting  No Results Available   
1413           NaN  Not yet recruiting  No Results Available   
1620     VID-WOMAN  Not yet recruiting  No Results Available   
1638     ANTICOVID  Not yet recruiting  No Results Available   
2376           NaN  Not yet recruiting  No Results Available   
2439  Pepcid4COV19  Not yet recruiting  No Results Available   
2592           NaN  Not yet recruiting  No Results Available   
2822           NaN  Not yet recruiting  No Results Available   
3066           NaN  Not yet recruiting  No Results Available   
3447           NaN  Not yet recruiting  No Results Available   
3471           NaN  Not yet recruiting  No Results Available   
3785           NaN  Not yet recruiting  No Results Available   
4469           NaN  Not yet recruiting  No Results Available   
5457      PPE-TEAM  Not yet recruiting  No Results Available   
5523           NaN  Not yet recruiting  No Results Available   
5651      OPERATOR  Not yet recruiting  No Results Available   

                                             Conditions  \
117                                            COVID-19   
176                                             Covid19   
278                                   COVID-19 Vaccines   
766                                           Pregnancy   
1413                                 Covid19|Human Milk   
1620                    Pregnancy Complications|Covid19   
1638                          Severe COVID-19 Pneumonia   
2376                             Overweight and Obesity   
2439                                            Covid19   
2592                                         SARS-CoV-2   
2822                                            Covid19   
3066                                            Covid19   
3447                                            Covid19   
3471                                            Covid19   
3785                        Covid19|Physical Disability   
4469                                    Vaccine Refusal   
5457  Safety Issues|Communication|Cardiopulmonary Ar...   
5523                           Autism Spectrum Disorder   
5651                                   Rena

#### Q4. Display the clinical trial with enrollments less than 80.

In [18]:
df[df['Enrollment'] < 80]

Rank   NCT Number                                              Title  \
1        2  NCT04595136  Study to Evaluate the Efficacy of COVID19-0001...   
4        5  NCT04395924         Maternal-foetal Transmission of SARS-Cov-2   
5        6  NCT04516954          Convalescent Plasma for COVID-19 Patients   
8        9  NCT04602884  Early Detection of COVID-19 Using Breath Analysis   
14      15  NCT04842708  Evaluation of Anti-COVID 19 Pfizer Vaccination...   
...    ...          ...                                                ...   
5760  5761  NCT03594734  Group Lifestyle Balance™ for Individuals With ...   
5764  5765  NCT03823469  Evaluating the Impact of a Culinary Coaching T...   
5766  5767  NCT04184895  A Study to Assess the Safety, Tolerability and...   
5774  5775  NCT04180709  CBT to Reduce Insomnia and Improve Social Reco...   
5775  5776  NCT04335643  Telehealth CBT for Adolescents and Young Adult...   

           Acronym                   Status         Study Results  \
1         COVID-19       Not yet recruiting  No Results Available   
4     TMF-COVID-19               Recruiting  No Results Available   
5             CPCP  Enrolling by invitation  No Results Available   
8         COVID-19                Suspended  No Results Available   
14        COVID-19               Recruiting  No Results Available   
...            ...                      ...                   ...   
5760           NaN   Active, not recruiting  No Results Available   
5764           NaN               Recruiting  No Results Available   
5766           NaN               Recruiting  No Results Available   
5774         CRISP               Recruiting  No Results Available   
5775          cSLE               Recruiting  No Results Available   

                                             Conditions  \
1                                  SARS-CoV-2 Infection   
4     Maternal Fetal Infection Transmission|COVID-19...   
5                                              COVID 19   
8                                               Covid19   
14                                              Covid19   
...                                                 ...   
5760                             Traumatic Brain Injury   
5764                                 Overweight|Obesity   
5766                       Allergic to House Dust Mites   
5774                Psychotic Disorders|Psychosis|Sleep   
5775                       Systemic Lupus Erythematosus   

                                          Interventions  \
1       Drug: Drug COVID19-0001-USR|Drug: normal saline   
4     Diagnostic Test: Diagnosis of SARS-Cov2 by RT-...   
5              Biological: Convalescent COVID 19 Plasma   
8     Diagnostic Test: Breath biopsy sampling using ...   
14         Diagnostic Test: vaccination against COVID19   
...                                                 ...   
5760  Behavioral: Group Lifestyle Balance™|Other: At...   
5764  Behavioral: CCTP|Behavioral: Nutritional couns...   
5766            Biological: ASP2390|Biological: Placebo   
5774                                    Device: Sleepio   
5775                                  Behavioral: TEACH   

                                       Outcome Measures  \
1     Change on viral load results from baseline aft...   
4     COVID-19 by positive PCR in cord blood and / o...   
5     Evaluate the safety|Change in requirement for ...   
8     Correlation between Volatile Organic Compounds...   
14    Association between breath VOCs and IgG in blo...   
...                                                 ...   
5760  Change in weight|Step Count|Waist and Arm Circ...   
5764  Body weight at 6 months|Body weight|Culinary a...   
5766  Number of participants with Adverse Events (AE...   
5774  Change from baseline Work and Social Adjustmen...   
5775  Recruitment rates of the study|Retention rates...   

                                  Sponsor/Collaborators  Gender  \
1                            United Medical Sp

#### Q5. Display the first 5 clinical trials where 'Study Type' is 'Observational.

In [ ]:
df[df['Study Type'] == 'Observational'].head()

Rank   NCT Number                                              Title  \
2      3  NCT04395482  Lung CT Scan Analysis of SARS-CoV2 Induced Lun...   
3      4  NCT04416061  The Role of a Private Hospital in Hong Kong Am...   
4      5  NCT04395924         Maternal-foetal Transmission of SARS-Cov-2   
7      8  NCT04634214  The Severity of COVID 19 in Diabetes and Non-d...   
11    12  NCT04412265          Frailty in Elderly Patients With COVID-19   

         Acronym                  Status         Study Results  \
2    TAC-COVID19              Recruiting  No Results Available   
3       COVID-19  Active, not recruiting  No Results Available   
4   TMF-COVID-19              Recruiting  No Results Available   
7        COVID19      Not yet recruiting  No Results Available   
11     FRA-COVID              Recruiting  No Results Available   

                                           Conditions  \
2                                             covid19   
3                                               COVID   
4   Maternal Fetal Infection Transmission|COVID-19...   
7                              Covid19|Type2 Diabetes   
11                                            Covid19   

                                        Interventions  \
2   Other: Lung CT scan analysis in COVID-19 patients   
3           Diagnostic Test: COVID 19 Diagnostic Test   
4   Diagnostic Test: Diagnosis of SARS-Cov2 by RT-...   
7                                                 NaN   
11  Other: Relation between frailty and clinical o...   

                                     Outcome Measures  \
2   A qualitative analysis of parenchymal lung dam...   
3   Proportion of asymptomatic subjects|Proportion...   
4   COVID-19 by positive PCR in cord blood and / o...   
7   Severity of COVID 19 among people with and wit...   
11  Development of a tool to measure frailty|A "pr...   

                                Sponsor/Collaborators  Gender  \
2                        University of Milano Bicocca     All   
3                     Hong Kong Sanatorium & Hospital     All   
4   Centre Hospitalier Régional d'Orléans|Centre d...  Female   
7   India Diabetes Research Foundation & Dr. A. Ra...     All   
11                       University of Milano Bicocca     All   

                                            Age Phases  Enrollment Funded Bys  \
2     18 Years and older   (Adult, Older Adult)    NaN       500.0      Other   
3                     Child, Adult, Older Adult    NaN      2500.0   Industry   
4                18 Years to 48 Years   (Adult)    NaN        50.0      Other   
7   18 Years to 99 Years   (Adult, Older Adult)    NaN      1050.0      Other   
11    18 Years and older   (Adult, Older Adult)    NaN       300.0      Other   

       Study Type                                      Study Designs  \
2   Observational  Observational Model: Cohort|Time Perspective: ...   
3   Observational  Observational Model: Cohort|Time Perspective: ...   
4   Observational  Observational Model: Cohort|Time Perspective: ...   
7   Observational  Observational Model: Case-Control|Time Perspec...   
11  Observational  Observational Model: Case-Only|Time Perspectiv...   

       Other IDs         Start Date Primary Completion Date  Completion Date  \
2    TAC-COVID19        May 7, 2020           June 15, 2021    June 15, 2021   
3     RC-2020-08       May 25, 2020           July 31, 2020  August 31, 2020   
4   CHRO-2020-10        May 5, 2020                May 2021         May 2021   
7     IDRFARH015  November 16, 2020       February 16, 2021     May 16, 2021   
11     FRA-COVID     April 16, 2020           March 1, 2021    March 1, 2021   

         First Posted Results First Posted Last Update Posted  \
2        May 20, 2020                  NaN   November 9, 2020   
3        June 4, 2020                  NaN       June 4, 2020   
4        May 20, 2020                  NaN       June 4, 2020   
7   November 18, 2020                  NaN  November 18, 2020   
11  